testdata/seeder.sh

```diff
@@ -4,17 +4,29 @@ echo "=============================================="
 echo "#              SEEDER SCRIPT                 #"
 echo "=============================================="
 
-# generated by get_token.sh
-personal_access_token=$(cat token.txt)
+#set gitlab base url
+base_url="http://13.229.65.123:10080/"
```

Comment:
Make the `base_url` a shell argument? 

E.g: `./seeder.sh http://localhost:10080`

Ensure to update `.travis.yml` too. Travis wants localhost:10080